BMM.Core/Translation/en/main.json

```diff
@@ -280,6 +280,7 @@
     "OptionAppVersionHeader": "App version",
     "AppInfoTitle": "App information",
     "AppInfoAppVersion": "App version: ",
+    "AppInfoManufacturer": "Manufacturer: ",
```

Comment:
Where is this being used?

Reply:
Profile Screen -> App Version. The translation was never added.